### DISCUSSION TITLE
When to release the new version with RocketMQ

### DISCUSSION THREAD

**Discussion by yujintang** [apache/apisix#6002] [Q&A]
请问带有RocketMQ log 收集的版本有计划什么时候发布吗？

**Comment by juzhiyuan** [comment]
Hi I notice RocketMQ Logger is supported last month, let's wait for the coming new release.

https://github.com/apache/apisix/commits/master/apisix/plugins/rocketmq-logger.lua

**Reply by WeCreos** [reply]
I'm afraid it would take awhile

**Reply by juzhiyuan** [reply]
> I'm afraid it would take awhile

Hi, I don't get your point 😅
